docs/upgrade/money-class.md

```diff
@@ -0,0 +1,430 @@
+-                         'value' => 0,
+-                         'message' => 'Price must be greater or equal to {{ compared_value }}',
+-                     ]),
++                     new NotNegativeMoneyAmount(['message' => 'Price must be greater or equal to zero']),
+                  ],
+              ])
+      // ...
++
++     /**
++      * @param \Shopsys\FrameworkBundle\Component\Money\Money $money
++      * @param \Symfony\Component\Form\FormBuilderInterface $moneyBuilder
++      * @return string
++      */
++     protected function transformMoneyToView(Money $money, FormBuilderInterface $moneyBuilder): string
++     {
++         foreach ($moneyBuilder->getModelTransformers() as $modelTransformer) {
++             /** @var \Symfony\Component\Form\DataTransformerInterface $modelTransformer */
++             $money = $modelTransformer->transform($money);
++         }
++         foreach ($moneyBuilder->getViewTransformers() as $viewTransformer) {
++             /** @var \Symfony\Component\Form\DataTransformerInterface $viewTransformer */
++             $money = $viewTransformer->transform($money);
++         }
++
++         return $money;
++     }
+  }
+```
+
+## Data in SettingValue Entities
+The [`Setting`](/packages/framework/src/Component/Setting/Setting.php) component is now able to save and load [`SettingValue`](/packages/framework/src/Component/Setting/SettingValue.php) entities with an instance of `Money` as a value.
+To make sure that monetary values that are currently saved as a decimal string will be loaded as a `Money` instance, you have to manually create [a database migration](/docs/introduction/database-migrations.md) that changes the `type` to `money` for all `SettingValue` entities holding monetary values that you added into your project.
+Previously, any of types `string`, `integer` or `float` might have been used to save a numeric value into the database.
+
+For example see the `up` method of migration class [`Version20190220101938`](/packages/framework/src/Migrations/Version20190220101938.php) that migrates the `freeTransportAndPaymentPriceLimit` setting values to the `Money` type:
+```php
+/**
+ * @param \Doctrine\DBAL\Schema\Schema $schema
+ */
+public function up(Schema $schema)
+{
+    $this->sql('UPDATE setting_values SET type = \'money\' WHERE name = \'freeTransportAndPaymentPriceLimit\' AND type != \'none\'');
+}
+```
+
+*Note: If you didn't add any new `SettingValue` entities that hold monetary values you can safely skip this step.*
+*The migration in `shopsys/framework` mentioned above will take care of changing the type of the limits for free transport and payment in your DB.*
+
+## Data Fixtures
+All monetary values in data fixtures should use `Money` as well.
+If you haven't altered the data fixtures in any way, you should be ready to go when you copy the data fixtures from `shopsys/project-base` in `v7.0.0` (as explained in the upgrade notes of [PR #854](https://github.com/shopsys/shopsys/pull/854)).
```

Comment:
Please change the link to upgrade instructions instead of PR. When I click into the PR, I don't see upgrade notes and I don't know where are they.

Reply:
You mean a link to https://github.com/shopsys/shopsys/pull/854/files#diff-da18024d2b6b8b4b2fafe94d18cb2866 or to https://github.com/shopsys/shopsys/blob/master/docs/upgrade/UPGRADE-v7.0.0.md#application ?

Reply:
Both are better than just link to PR.... The first is the best in my opinion (even it is just a diff, it shows exactly what to to)

Reply:
Changed in 86c645ccc